docs/docsite/rst/installation_guide/intro_installation.rst

```diff
@@ -115,6 +115,10 @@ You may need to perform some additional configuration before you are able to run
 Installing Ansible with ``pip``
 -------------------------------
 
+.. note::
+
+	If you have Ansible 2.9 or older installed, you need to use ``pip uninstall ansible`` first to remove older versions of Ansible before re-installing it. Similarly, if you have installed Ansible 3, or ``ansible-base`` on its own, you need to use ``pip uninstall ansible-base`` before upgrading Ansible.
```

Comment:
This seems repetitive (and yeah, I started this trend in my last PR, but I'm rethinking it now). I'd vote that we make a single note about "Follow these instructions to install Ansible for the first time. If you have installed Ansible before, please see Upgrading for details on installing more recent versions", then use that everywhere we want people to know about uninstall/reinstall. That way if we need to update information about upgrading from version x to version y or version y to version z, we only need to do it in one place.